Configuration/PyReleaseValidation/scripts/das-up-to-nevents.py

```diff
@@ -147,19 +148,23 @@ def no_intersection():
 
         ## if we have access to eos we get from there ...
         if os.path.isdir(cert_path):
-            json_list = os.listdir(cert_path)
-            if len(json_list) == 0:
+            json_list_full = os.listdir(cert_path)
+            if len(json_list_full) == 0:
                 web_fallback == True 
-            json_list = [c for c in json_list if "golden" in c.lower() and "era" not in c.lower() and "ppref" not in c.lower()]
-            json_list = [c for c in json_list if c.lower().startswith("cert_c") and c.endswith("json")]
         else:
             web_fallback = True
         ## ... if not we go to the website
         if web_fallback:
             cert_url = base_cert_url + cert_type + "/"
-            json_list = get_url_clean(cert_url).split("\n")
-            json_list = [c for c in json_list if "golden" in c.lower() and "era" not in c.lower() and "cert_c" in c.lower() and "ppref" not in c.lower()]
-            json_list = [[cc for cc in c.split(" ") if cc.lower().startswith("cert_c") and cc.endswith("json")][0] for c in json_list]
+            json_list_full = get_url_clean(cert_url).split("\n")
+        pattern = re.compile("(cert_collisions\d{4}_\d*_\d*_golden.json)$", re.IGNORECASE)
```

Comment:
Maybe this regex could be directly `cert_collisions\d{4}_\d{6}_\d{6}_golden.json` ?

Reply:
Right!